### ISSUE TITLE
Annotate SV_type column with inversion/deletion/duplication/insertion

### ISSUE THREAD

**Issue by xiucz** [lgmgeo/AnnotSV#118]
Hi,

Are you interested in reannotating SV_type column with `inversion/deletion/duplication/insertion`, not just `BND`?

https://github.com/walaj/svaba/issues/4
Best,
xiucz

**Comment by lgmgeo** [comment]
Hi, 

Sorry, no plan in AnnotSV to re-classify BND as INV/DEL/DUP/INS.
I'm adding this feature to my todo list, but it won't be for this year.

Best,
Véronique

**Comment by minn333** [comment]
Dear Igmgeo:
         For BND, there is no ranking score result, isn't it? I found that, for structure variant calls from svABA, there are no ranking scores result after annotating with AnnotSV.

Thanks.

**Comment by lgmgeo** [comment]
@minn333 ,

You're right, no way to rank BND at this time as the ranking follows the ACMG/ClinGen guidelines for Gain and Loss only. It should be improved in 2023. Sorry for the delay.

Best,
Véronique

**Comment by minn333** [comment]
Thanks for your work! With best wishes,段德敏  從 Windows 的郵件傳送   寄件者: Geoffroy Véronique傳送時間: 2022年10月3日 下午 02:23收件者: lgmgeo/AnnotSV副本: minn333; Mention主旨: Re: [lgmgeo/AnnotSV] Annotate SV_type column with inversion/deletion/duplication/insertion (Issue ***@***.*** ,You're right, no way to rank BND at this time as the ranking follows the ACMG/ClinGen guidelines for Gain and Loss only. It should be improved in 2023. Sorry for the delay.Best,Véronique—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>

**Comment by minn333** [comment]
Thank you.  With best wishes,段德敏  從 Windows 的郵件傳送   寄件者: Geoffroy Véronique傳送時間: 2023年3月27日 下午 09:53收件者: lgmgeo/AnnotSV副本: minn333; Mention主旨: Re: [lgmgeo/AnnotSV] Annotate SV_type column with inversion/deletion/duplication/insertion (Issue #118) Closed #118 as completed.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
